README.md

```diff
@@ -14,3 +14,59 @@ This app can work offline on desktop, tablet and mobile.
 Data is stored locally on device in [PouchDB](https://pouchdb.com/) database and can be synced to the cloud.
 
 Implemented with [React](https://github.com/facebook/react) and [Semantic UI](https://github.com/Semantic-Org/Semantic-UI-React)
+
+## License
+Our software is distributed under the MIT License.
+
+## Installation
+
+### Prerequisites
+- npm, which you can install by running: 
+
+    ```npm install npm@latest –g```
+
+### How to install
+* Clone the repo from this website, then
+* In command line:
+  * ```$ git clone https://github.com/ayastreb/money-tracker.git```
+  * ```$ cd money-tracker```
+  * ```$ npm install react-scripts```
```

Comment:
I guess it can be just `npm install` or even `npm ci`

Reply:
Sounds good. I'll update that. Thanks!